src/List.tsx

```diff
@@ -240,6 +242,7 @@ class List<Value = string> extends React.Component<IProps<Value>> {
       clientY - this.state.initialY,
       this.props.lockVertically ? 0 : clientX - this.state.initialX
     );
+    this.props.onMove && this.props.onMove({ clientX, clientY });
```

Comment:
Maybe this should be also `ClientRect` instead of just coordinates?

Reply:
Depends on what we want to share – mouse position, the dragged item position or even both. Guess if we go with the client rect, we can trigger this even when using the keyboard shortcuts as well.

Reply:
Right. I suspect that ClientRect is more useful. For example, if I want other elements on the page to interact (reposition? change color?) with the dragged item, I'll need the position / dimensions of the dragged item. Also, whenever I would use mouse coordinates, the center of gravity is probably better choice anyway (like this example https://miksu.cz/londondnd/#39)